lib/middleware/token.js

```diff
@@ -52,15 +54,15 @@ var TokenError = require('../errors/tokenerror');
  */
 module.exports = function token(server, options) {
   options = options || {};
-  
-  if (!server) { throw new TypeError('oauth2orize.token middleware requires a server argument'); }
-  
+
+  if (!server) { throw new TypeError(g.f('{{oauth2orize.token}} middleware requires a server argument')); }
```

Comment:
lets translate to French for example 
`requires an argument server`
will be
`nécessite un argument de serveur` -> It makes no sense to me if the translation is 
`nécessite un 'server' d' argument` -> which is not even a sentence